subsystem/cluster_resource_rosa_test.go

```diff
@@ -148,6 +148,7 @@ var _ = Describe("Cluster creation", func() {
 		    cloud_region   = "us-west-1"
 			aws_account_id = "123"
 			sts = {
+				oidc_config = {},
```

Comment:
after you'll change it to "oidc_config_id" you'll no need an empty block anymore